pegasus/routes/hoc_routes.rb

```diff
@@ -25,10 +25,19 @@
 
   # set company to nil if not a valid company
   company = request.GET['company']
-  unless company.nil?
+  # Our partnership with cartoon network piggy-backs on our company profile.
+  # Pass their company param through to the congrats page even if no entry exists
+  # in the forms table.
+  unless company.nil? || company == 'CN'
```

Comment:
"Pass through the company param to the congrats page only if an entry exists in the forms or for the special case of cartoon network, where we need the company to add a customized link"